src/networks.js

```diff
@@ -10,37 +11,29 @@ export const RINKEBY_USABILITY_COURT =
 export const RINKEBY_STAGING_COURT =
   '0x52180Af656A1923024D1ACcF1D827AB85cE48878'
 
-export const networks = {
+export const networkConfigs = {
   main: {
     court: '0xee4650cBe7a2B23701D416f58b41D8B76b617797',
```

Comment:
If it's not too much work, it would be great to key these under a `addresses` key like we have in [the client](https://github.com/aragon/aragon/blob/master/src/network-config.js#L13).

Reply:
is it ok for you if i change this in a separated pr only for that?

Reply:
Yep—let's do it!